doc/source/user_guide/404_page.rst

```diff
@@ -42,19 +42,26 @@ your ``conf.py`` file:
 Configure your 404 page
 -----------------------
 You can use the default 404 page that the ``ansys-sphinx-theme`` package supplies
-or create and use a custom 404 page. 
+or create and use a custom 404 page.
 
 Use the default 404 page
 ~~~~~~~~~~~~~~~~~~~~~~~~
-To use the default 404 page, add the following lines in the ``conf.py`` file:
+To use the default 404 page, you can use the ``generate_404`` function in the
+``ansys_sphinx_theme`` module to create and use a custom cover page:
 
-.. code-block:: 
+.. code-block:: python
 
-    from ansys_sphinx_theme import page_404
+    from ansys_sphinx_theme import generate_404
 
 
     # Configure sphinx-notfound-page
-    notfound_template = page_404
+    notfound_context = {
+    'body': generate_404(<issue_page_of_project to direct>,
```

Comment:
In line with what @RobPasMue said, remember to remove this argument in the documentation too, as it can be computed inside of the function.